.travis.yml

```diff
@@ -25,16 +25,21 @@ env:
   - VERSION="11.0" TESTS="0" LINT_CHECK="0"
   matrix:
   - LINT_CHECK="1"
-  - TESTS="1" ODOO_REPO="OCA/OCB"
-  - TESTS="1" ODOO_REPO="odoo/odoo"
+  - TESTS="1" ODOO_REPO="OCA/OCB" EXCLUDE="runbot_buildout"
+  - TESTS="1" ODOO_REPO="OCA/OCB" EXCLUDE="runbot_travis2docker"
 
 
 install:
+  # only install extra dependencies when we're not linting
+  - pip install zc.recipe.egg --no-cache-dir
+  # the latest package on pip doesn't speak python3
+  - pip install https://github.com/anybox/anybox.recipe.odoo/tarball/master/anybox.recipe.odoo-1.9.3.dev0.tar.gz
```

Comment:
Is not good use requirements.txt for this case, right?

Reply:
I tried to pin this to a git+https url, but that didn't work the first couple of tries and then I figured I'll just wait it out until this version is released